### ISSUE TITLE
whitelist embeds from streamable.com

### ISSUE THREAD

**Issue by kareila** [dreamwidth/dw-free#2377]
https://www.dreamwidth.org/support/see_request?id=39291

Sample code:
```
<div style="width: 100%; height: 0px; position: relative; padding-bottom: 56.250%;"><iframe src="https://streamable.com/s/asq5b/knxvuf" frameborder="0" width="100%" height="100%" allowfullscreen style="width: 100%; height: 100%; position: absolute;"></iframe></div>
```

**Comment by rshatch** [comment]
claiming

**Comment by momijizukamori** [comment]
Fixed via #2407
